src/styles/main.scss

```
@@ -1,3 +1,58 @@
-body {
-  margin: 0;
+@import "./reset";
+@import "./tools/variables";
+@import "./tools/mixin";
+
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap-size;
+  padding-top: 40px;
+
+  &--start-day {
+    @each $day, $day-count in $days-in-week {
+      &--#{$day} :first-child {
+        margin-left: ($day-count) * $day-margin;
+      }
+    }
+  }
+
+  &__day {
+    font-style: arial, helvetica;
+    font-size: 30px;
```

```suggestion
    font-style: arial, helvetica;
    font-size: 30px;
```
Move it to total styles, into normalize file